app/build.gradle

```diff
@@ -125,6 +127,9 @@ android {
     }
 
     sourceSets {
+        main {
+            java.srcDirs = ['src/main/java']
+        }
```

Comment:
Why is this needed? Doesn't this source set exist by default? Is spotbugs not able to find the default source sets? Please add a comment here explaining why we do this. :)

Reply:
Spotbugs is able to find the default source sets.